src/AppInstallerRepositoryCore/CompositeSource.h

```diff
@@ -57,7 +52,7 @@ namespace AppInstaller::Repository
         SearchResult SearchAvailable(const SearchRequest& request) const;
 
         std::shared_ptr<ISource> m_installedSource;

```

Comment:
>std::shared_ptr<ISource> [](http://example.com/codeflow?start=8&length=24)

Should this one be updated to use Source too since you are updating the available ones already?